src/components/util/virtualized-table.js

```diff
@@ -247,12 +301,16 @@ class MuiVirtualizedTable extends React.PureComponent {
             return index < reorderedIndex.length ? reorderedIndex[index] : 0;
         };
 
+        const sizes = this.sizes(this.props.columns);
         return (
             <AutoSizer>
                 {({ height, width }) => (
                     <Table
                         height={height}
-                        width={width}
+                        width={Math.max(
```

Comment:
This is bad because it overflows the parent:
![Capture d’écran de 2021-03-31 16-46-27](https://user-images.githubusercontent.com/89208/113163680-aba1c500-9240-11eb-9c63-fd7af9ad75aa.png)